src/lib/frontend/models.ml

```diff
@@ -312,10 +312,19 @@ module SmtlibCounterExample = struct
     | [Ty.Trecord _r, _arg] -> begin
         match xs_values with
         | [record_name,_] ->
+          (* HOTFIX: this fix is temporary. The current implementation of
```

Comment:
Sure, actually it will be fix by the new representation of model values because we will use `Expr.t` here instead of `string`.

Reply:
It's done #958.

Reply:
The comment should mention a link to #958 — the point of having an issue is so that we can track what we have to do, and the point of having the link in the comment is so that we know which issue to update if the problem ends up fixed by an unrelated refactor.

Reply:
Done :)